9.3/paper.tex

```diff
@@ -1089,7 +1100,11 @@ \subsection{New and improved tutorials and code gallery programs}
   or alternatively as the second derivative of an energy functional
   that then also yields the residual itself.
 
-\item \texttt{step-74} SIPG for the Poisson equation \todo[inline]{TODO}
+\item \texttt{step-74} implements the symmetric interior penalty Galerkin 
+(SIPG) method for Poisson's equation using the \texttt{FEInterfaceValues} class
+in combination with \texttt{MeshWorker::mesh\_loop()}. This tutorial demonstrates a
+simple and mesh-independent way to assemble cell face integrals.
```

Comment:
```suggestion
simple and mesh-independent way to assemble cell and face integrals.
```
?

Reply:
I wanted to say integrals of cell faces, not including cell integrals. Maybe just use face integrals or surface integrals?

Reply:
Just call it face integrals.